src/main/java/org/disq_bio/disq/impl/formats/vcf/VcfSource.java

```diff
@@ -96,14 +97,18 @@ private static InputStream bufferAndDecompressIfNecessary(final InputStream in)
     }
     enableBGZFCodecs(conf);
 
-    Broadcast<VCFCodec> vcfCodecBroadcast = jsc.broadcast(getVCFCodec(jsc, path));
+    VCFCodec vcfCodec = getVCFCodec(jsc, path);
+    VCFHeader header = vcfCodec.getHeader();
```

Comment:
Could you add a comment explaining why this is necessary for future generations?  The version thing is very non obvious.

Reply:
Done